src/pages-and-resources/proctoring/Settings.test.jsx

```diff
@@ -687,11 +700,19 @@ describe('ProctoredExamSettings', () => {
 
     it('Successfully updates exam configuration and studio provider is set to "lti_external" for lti providers', async () => {
       await act(async () => render(intlWrapper(<IntlProctoredExamSettings {...defaultProps} />)));
-      // Make a change to the provider to proctortrack and set the email
+      // Make a change to the provider to test_lti and set the email
```

Comment:
This looks like some copypasta, and the email was never set, so I've fixed that.